docs/how-to/h-async-replication.md

```diff
@@ -0,0 +1,321 @@
+# Configuring asynchronous replication: MySQL InnoDB ClusterSet
+
+> **WARNING**: it is an internal article. Do NOT use it in production! Contact [Canonical Data Platform team](https://chat.charmhub.io/charmhub/channels/data-platform) if you are interested in the topic.
+
+## Intro
+Read more about [MySQL InnoDB ClusterSet](https://dev.mysql.com/doc/mysql-shell/8.0/en/innodb-clusterset.html) and watch the [presentation](https://videohub.oracle.com/media/Disaster+Recovery+with+MySQL+InnoDB+ClusterSet+-+What+is+it+and+how+do+I+use+itF/1_w0naxll1/170883371).
+
+For simplicity, we deploy both MySQL InnoDB Clusters into the same model, but the proper design is to install them in different Regions/Clouds:
+
+![MySQL_async_ClusterSet|690x586](upload://t8RCVAmkq1Hp5BwJM85ZJwxkREP.png) 
+
+## Bootstrap
+Bootstrap two MySQL InnoDB Clusters (Rome and Lisbon) and test application to generate SQL traffic:
+```shell
+juju deploy --constraints mem=2G mysql mysql-1st --channel 8.0/edge -n 3 --config cluster-name=ROME
+juju deploy --constraints mem=2G mysql mysql-2nd --channel 8.0/edge -n 3 --config cluster-name=LISBON
+juju deploy mysql-test-app --channel edge
+juju relate mysql-1st mysql-test-app:database
+juju wait-for application mysql-1st --query='status=="active"' && juju wait-for application mysql-2nd --query='status=="active"'
+```
+## Collect clusters details
+Collect both clusters credentials to reuse later:
+```shell
+cluster_name_1st=$(juju show-unit mysql-1st/0 | yq '.. | select(. | has("cluster-name")).cluster-name')
+cluster_name_2nd=$(juju show-unit mysql-2nd/0 | yq '.. | select(. | has("cluster-name")).cluster-name')
```

Comment:
2nd cluster credentials never needed

Reply:
Correct, they never needed if everything went fine!
They were never available here initially... then on the final testing things went wrong, and one node didn't join the cluster... and I had old credentials for those node anywhere anymore... so decided to store all credentials here to handle exceptions (if any).